src/components/App.js

```diff
@@ -47,9 +47,28 @@ function App() {
     setDoc(doc(db, `${storeId}`, "fishes"), { ...fishes, ...sampleFishes});
   };
 
-  const addToOrder = (key) => {
+  const updateFish = ( key, updatedFish ) => {
+    const copyFishes = { ...fishes };
```

Comment:
Maybe you can rename it to `oldFishes` or something like that?

Reply:
Ok. I understand.